cluster/src/main/scala/io/snappydata/impl/LeadImpl.scala

```diff
@@ -239,6 +240,22 @@ class LeadImpl extends ServerImpl with Lead
     }
   }
 
+  private def checkAuthProvider(props: Properties): Unit = {
+    var authP = props.getProperty(Attribute.AUTH_PROVIDER)
+    if (authP == null) {
+      authP = props.getProperty(Attribute.SERVER_AUTH_PROVIDER)
+    }
```

Comment:
I'm not sure I understood.
If AUTH_PROVIDER is specified, it should override SERVER_AUTH_PROVIDER.
Do you mean to suggest that if both are non-null and different then throw error?

Actually, for snappydata we should altogether skip checking for SERVER_AUTH_PROVIDER as we are not mentioning it in snappydata docs.